q2_feature_table/_summarize/summarize_assets/sample-frequency-detail.html

```diff
@@ -1,35 +1,205 @@
+    <div id="plot">
+      <div id="loading" class="spinner">
+        <div class="rect1"></div>
+        <div class="rect2"></div>
+        <div class="rect3"></div>
+        <div class="rect4"></div>
+        <div class="rect5"></div>
+      </div>
+    </div>
+  </div>
+
+  <div class="col-lg-4">
+    <div class="row">
+      <div class="col-lg-12"><h3>Plot Controls</h3></div>
+    </div>
+
+    <div class="row"><div class="col-lg-12" id="toolbar"></div></div>
+
+    <div class="row">
+      <div class="col-lg-12" id="metadata-category">Metadata Category</div>
+    </div>
+
+    <div class="row">
+      <div class="col-lg-12" id="sampling-depth-slider"></div>
+    </div>
 
-{% block content %}
-  <div class="row">
-    <div class="form-group col-lg-2 col-md-2 col-sm-12 col-xs-12">
-      <label for="slider-value">Sampling depth:</label>
-      <input
-        id="slider-value"
-        class="form-control"
-        type="number"
-        min="0"
-      >
-      <div>(Zero implies no even sampling.)</div>
-      <br>
-      <input
-        id="slider"
-        type="range"
-        value="0"
-        min="0"
-        max="{{ max_count }}"
-      >
-      <br>
+    <div class="row">
+      <div class="col-lg-12" id="text_field">To be updated text!</div>
     </div>
-    <div id="histogram" class="col-lg-10 col-md-10 col-sm-12 col-xs-12"></div>
   </div>
-  <div class="row">
-    <div id="table" class="col-lg-10 col-md-10 col-sm-12 col-xs-12 pull-right"></div>
+</div>
+
+{% include 'js-error-handler.html' %}
+<script id="spec" type="application/json">
+  {{ vega_spec }}
+</script>
+<script type="text/javascript">
+  $(document).ready(function() {
+    var spec = JSON.parse(document.getElementById("spec").innerHTML);
+
+    // Try and come up with a good initial estimate of plot dimensions,
+    // based on the browser dimensions.
+    var opts = {
+      width: $("#plot").width(),
+      height: $("#plot").width() / 2
+    };
+
+    vegaEmbed("#plot", spec, opts)
+      .then(function(result) {
+        result.view.logLevel(vega.Warn);
+        // Stash the view object in global namespace for debugging purposes.
+        // Check out https://vega.github.io/vega/docs/api/debugging/
+        // for more details.
+        window.v = result.view;
+      })
+      .catch(function(error) {
+        // From 'js-error-handler.html'
+        handleErrors([error], $("#plot"));
+      });
+
+    // Clean up the vega-embed toolbar
+    var toolbar = $("div .vega-actions").detach();
+    toolbar.addClass("btn-group");
+    toolbar.children("a").each(function() {
+      $(this).addClass("btn btn-default");
+    });
+
+    toolbar.appendTo("#toolbar");
+
+    var category = $("#metadata-category .vega-bind");
+    category
+      .children(".vega-bind-name")
+      .replaceWith("<label> Metadata Category &nbsp;</label>");
+    category.children("select").addClass("form-control");
+
+    var sampling_depth_slider = $("#sampling-depth-slider .vega-bind");
```

Comment:
JS variables in camel case.

```suggestion
    var samplingDepthSlider = $("#sampling-depth-slider .vega-bind");
```